content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+
+### remove spots and unwanted objects
+
+Use the [_retouch_](../../module-reference/processing-modules/retouch.md) module to remove unwanted objects by replacing pixels with detail from elsewhere in the image. This module also offers powerful techniques for removing large-scale objects (such as spots or blemishes) while leaving fine-scale details (like hairs and follicles) intact. The most common use for this module is to remove dust spots from images or blemishes from skin.
+
+### remove atmospheric haze
+
+There are two methods for removing atmospheric haze in darktable -- you can either use the [haze removal](../../module-reference/processing-modules/haze-removal.md) module or the "dehaze" preset in the [_diffuse or sharpen_](../../module-reference/processing-modules/diffuse.md) module.
+
+## creative adjustments
+
+### crop and frame
+
+Use the [_crop_](../../module-reference/processing-modules/crop.md) module to crop your image and the [_framing_](../../module-reference/processing-modules/framing.md) module to surround your image with a colored frame. Both modules can be set to use a predefined or custom aspect ratio -- for example, you could place a square-cropped image into a 3:2 frame.
+
+### dodge and burn
+
+Dodging and burning is a traditional darkroom technique to add and remove brightness from an image. There are two recommended ways to achieve this
+
+- If you want to selectively dodge or burn only certain objects you can apply a [new instance](../../darkroom/processing-modules/multiple-instances.md) of the [_exposure_](../../module-reference/processing-modules/exposure.md) module using a [drawn mask](../../darkroom/masking-and-blending/masks/drawn.md) to isolate the effect to the required area of the image (see also the [mask refinement](../../darkroom/masking-and-blending/masks/refinement-controls.md) section for more information). Move the exposure slider to alter the brightness of the masked area.
+- If you wish to dodge or burn areas with a similar brightness (for example, to brighten the shadows or darken the highlights) you should use the [_tone equalizer_](../../module-reference/processing-modules/tone-equalizer.md) module.
+
+### convert to monochrome
+
+darktable provides a number of ways to remove the color from your image. The most flexible method is to use the [_gray tab of the color calibration module_](../../module-reference/processing-modules/color-calibration.md#gray-tab). A number of film emulation presets are available in this module to provide you with a starting point.
+
+See the [developing monochrome images](../../guides-tutorials/monochrome.md) section for details of other techniques.
+
+### color grading
+
+The [_color balance rgb_](../../module-reference/processing-modules/color-balance-rgb.md) module is your one-stop-shop for controlling the colors in your image. Adjustments can be isolated to the shadows, highlights and mid-tones, or applied at a global level.
```

Comment:
I feel there should be more info here, or at least some reference where the reader can explore further what is possible.

Reply:
you mean like a link to the module reference documentation? If you want to add something after merge that's fine but I'm still not yet familiar with this module